src/content/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration.mdx

```diff
@@ -1850,147 +1850,146 @@ To set permanent data storage on Kubernetes, the user has two options:
 
-A Kubernetes Job age of 5 minutes (`kubectl get jobs -n YOUR_NAMESPACE`) is a conservative target to account for variability in how long it takes pods to complete and how many synthetics jobs need to run per minute (jobs rate). The following equations can be used as a starting point for `completions` and `parallelism` for each runtime. Adjustments may need to be made based on observations of private location queue growth.
+A key consideration when sizing your runtimes is that a single SJM instance has a maximum throughput of **approximately 15 heavyweight jobs per minute** (scripted API and browser monitors). This is due to an internal threading strategy that favors the efficient competition of jobs across multiple SJMs over the raw number of jobs processed per SJM.
 
-```math
-completions = 300 / avg job duration (s)
-parallelism = synthetics jobs per 5 minutes / completions
-```
+You can use your average job duration to calculate the maximum effective `parallelism` for a single SJM before hitting this throughput ceiling:
+
+$$Parallelism_{max} \approx 15 \times \mathrm{ajd}_m$$
+
+Where $\mathrm{ajd}_m$ is the average job duration in minutes.
+
+If your monitoring needs exceed this \~15 jobs/minute limit, you must **scale out** by deploying multiple SJM instances. You can [check if your job queue is growing](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-maintenance-monitoring/) to see if more instances are needed.
+
+The `parallelism` setting controls how many pods of a particular runtime run concurrently, and it is the equivalent of the `HEAVYWEIGHT_WORKERS` environment variable in the Docker and Podman SJM. The `completions` setting controls how many pods of a particular runtime must complete before the `CronJob` can start another Kubernetes Job for that runtime. For improved efficiency, `completions` should be set to 6-10x the `parallelism` value.
+
+The following equations can be used as a starting point for `completions` and `parallelism` for each runtime.
+
+$$Completions = \frac{300}{ajd_s}$$
+
+Where $\mathrm{ajd}_s$ is the average job duration in seconds.
 
-Different runtimes will likely have different synthetics job durations and rates. The following queries can be used to obtain average duration and rate for a private location.
+$$Parallelism = \frac{sj_5m}{Completions}$$
+
+Where $\mathrm{sj_5m}$ is the number of synthetics jobs per 5 minutes.
```

Comment:
$$\mathrm{Parallelism} = \frac{N_{\mathrm{5m}}}{\mathrm{Completions}}$$

Where $N_{\mathrm{5m}}$ is the **number** of synthetics jobs per 5 minutes.